bots/activity.bot.js

```diff
@@ -0,0 +1,84 @@
+const _ = require('lodash');
+const config = require('pico-conf');
+const Slack = require('pico-slack');
+
+
+const DEBUG = config.get('activitybot.debug');
+const TARGET_CHANNEL = config.get('activitybot.target_channel');
+
+const MESSAGE_COUNT_THRESHOLD = parseInt(
+	config.get('activitybot.threshold.message_count')
+);
```

Comment:
Don't forget to either include `10` as a second argument here, or just use `_.parseInt` instead.